server/models/Books.js

```diff
@@ -0,0 +1,15 @@
+const mongoose = require("mongoose");
+mongoose.Promise = global.Promise;
+
+const Schema = mongoose.Schema;
+
+const novelSchema = new mongoose.Schema ({
+  bookTitle: String,
+  bookChapter: String,  //can be chapter name or chapter number
+  bookPath: String  //URL where the file is stored in public folder
+},
```

Comment:
Could be best to keeo consistency with either calling these fields `novelTitle`, `novelPath` etc. or just call them `title` etc. as they will always be referenced within the context of novels.

Reply:
renamed
![renamed vars](https://user-images.githubusercontent.com/42120918/109331200-775b7300-7854-11eb-8b95-18a537f3a8b9.JPG)